testing_app/test_driver/e2e_test.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:e2e/e2e_driver.dart' as driver;
+
+Future<void> main() => driver.e2eDriver(responseDataCallback: (data) async {
+      await driver.writeResponseData(
+        data['scrolling'] as Map<String, dynamic>,
```

Comment:
Are these values also written to stdout, so they're visible in CI logs?

Reply:
By default no, but I can add a print.

Reply:
Done